packages/gatsby-theme-blog/src/components/posts.js

```diff
@@ -1,42 +1,15 @@
-import React, { Fragment } from "react"
-import { Link } from "gatsby"
-import { Styled, css } from "theme-ui"
+import React from "react"
 
-import Layout from "../components/layout"
-import SEO from "../components/seo"
-import Footer from "../components/home-footer"
+import Layout from "./layout"
+import SEO from "./seo"
+import Footer from "./home-footer"
+import PostList from "./post-list"
 
 const Posts = ({ location, posts, siteTitle, socialLinks }) => (
   <Layout location={location} title={siteTitle}>
+    <SEO title="Home" />
     <main>
-      {posts.map(({ node }) => {
-        const title = node.title || node.slug
-        const keywords = node.keywords || []
-        return (
-          <Fragment key={node.slug}>
-            <SEO title="Home" keywords={keywords} />
```

Comment:
I wasn't 100% sure if this was functionality we wanted to keep or not? I imagine if we want to aggregate all post keywords for the post index we could reduce them and pass along to line 10 above?

Reply:
I'm not sure what's better for SEO, but maybe we should ask before making this change